webui/react/src/stores/cluster.tsx

```diff
@@ -206,10 +231,19 @@ class ClusterStore extends PollingStore {
 
   public async poll() {
     const agentRequest = getAgents({}, { signal: this.canceler?.signal });
+    const agentWithSlotsRequest = getAgents(
+      { excludeSlots: true },
```

Comment:
shouldn't this be false?

Reply:
question: I can't tell if we're doing this or not but the point is check and to not request slots when we have many agents and or slots otherwise web hits the same perf issues.

have you been able to test with the scaled up agents to see the effect?

Reply:
right, should be false. yeah this should be handled now